src/commands/init/ai-rules.ts

```diff
@@ -0,0 +1,447 @@
+  const configs: Record<string, string> = {
+    vscode: JSON.stringify(
+      {
+        servers: {
+          netlify: {
+            type: 'stdio',
+            command: 'npx',
+            args: ['-y', '@netlify/mcp'],
+          },
+        },
+      },
+      null,
+      2,
+    ),
+    cursor: JSON.stringify(
+      {
+        mcpServers: {
+          netlify: {
+            command: 'npx',
+            args: ['-y', '@netlify/mcp'],
+          },
+        },
+      },
+      null,
+      2,
+    ),
+    windsurf: JSON.stringify(
+      {
+        mcpServers: {
+          netlify: {
+            command: 'npx',
+            args: ['-y', '@netlify/mcp'],
+          },
+        },
+      },
+      null,
+      2,
+    ),
+  }
+
+  return (
+    configs[ide.key] ||
+    JSON.stringify(
+      {
+        mcpServers: {
+          netlify: {
+            command: 'npx',
+            args: ['-y', '@netlify/mcp'],
+          },
+        },
+      },
+      null,
+      2,
+    )
+  )
+}
+
+// VS Code specific MCP configuration
+const configureMcpForVSCode = async (config: string, projectPath: string): Promise<void> => {
+  const configPath = resolve(projectPath, '.vscode', 'mcp.json')
+
+  try {
+    // Create .vscode directory if it doesn't exist
+    await fs.mkdir(resolve(projectPath, '.vscode'), { recursive: true })
+
+    // Write or update mcp.json
+    let existingConfig: Record<string, unknown> = {}
+    try {
+      const existingContent = await fs.readFile(configPath, 'utf-8')
+      existingConfig = JSON.parse(existingContent) as Record<string, unknown>
+    } catch {
+      // File doesn't exist or is invalid JSON
+    }
+
+    const mcpConfig = JSON.parse(config) as Record<string, unknown>
+    const updatedConfig = { ...existingConfig, ...mcpConfig }
+
+    await fs.writeFile(configPath, JSON.stringify(updatedConfig, null, 2), 'utf-8')
+    log(`${chalk.green('✅')} VS Code MCP configuration saved to ${chalk.cyan('.vscode/mcp.json')}`)
+  } catch (error) {
+    throw new Error(`Failed to configure VS Code MCP: ${error instanceof Error ? error.message : 'Unknown error'}`)
+  }
+}
+
+// Cursor specific MCP configuration
+const configureMcpForCursor = async (config: string, projectPath: string): Promise<void> => {
+  const configPath = resolve(projectPath, '.cursor', 'mcp.json')
+
+  try {
+    await fs.mkdir(resolve(projectPath, '.cursor'), { recursive: true })
+    await fs.writeFile(configPath, config, 'utf-8')
+    log(`${chalk.green('✅')} Cursor MCP configuration saved to ${chalk.cyan('.cursor/mcp.json')}`)
+  } catch (error) {
+    throw new Error(`Failed to configure Cursor MCP: ${error instanceof Error ? error.message : 'Unknown error'}`)
+  }
+}
+
+// Windsurf specific MCP configuration
+const configureMcpForWindsurf = async (config: string, _projectPath: string): Promise<void> => {
+  const { homedir } = await import('node:os')
+  const configPath = resolve(homedir(), '.codeium', 'windsurf', 'mcp_config.json')
```

Comment:
nit: Save the path to the `windsurf` directory in a variable to avoid resolving the path multiple times.